packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -236,6 +242,38 @@ export class FeatureDevController {
                         },
                     ],
                 })
+            } else if (err instanceof PlanIterationLimitError) {
+                this.messenger.sendErrorMessage(err.message, message.tabID, this.retriesRemaining(session))
+                this.messenger.sendAnswer({
+                    type: 'system-prompt',
+                    tabID: message.tabID,
+                    followUps: [
+                        {
+                            pillText: 'Discuss a new plan',
```

Comment:
Does this end the current conversation for the customer? They may not understand the implications of clicking this.

Reply:
Yes, it ends the current conversation for the customer and will and start a new one when they request for a new plan.